src/core/RadioButton/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import useStyles from './styles';
+import Radio, { RadioProps } from '@material-ui/core/Radio';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+
+const RadioButton: React.FC<RadioProps> = (props) => {
+  const classes = useStyles();
+
+  return (
+    <div>
```

Comment:
This doesn't need to be wrapped in a div

Reply:
> Can you put an example in storybook where I have multiple radio buttons (say 3) and in
> case 1: I should be able to select only one of the 3
> case 2: I an select multiple among the 3

So, multiselection must be allowed via props or there is some other way? 
It's now this way on my end. I've checked MUI, and for it to be multiselected, radio buttons must be in radio group. 
Should the radio group tags be outside of the component or inside? 

`<ThemedBackground platform="litmus-portal" row>
      <RadioGroup row>
        <RadioButton value="1">Target cluster</RadioButton>
        <RadioButton disabled={true} checked={true}>
          Target cluster
        </RadioButton>
      </RadioGroup>
    </ThemedBackground>`

Reply:
yes perfect it can be in storybook @saten16